autogen/templates/md_led-color-methods.liquid

```diff
@@ -0,0 +1,8 @@
+Method name | Return type | Arguments | Description
+---|---|---|---
+{% for ledColor in classes.led.ev3LedColors
+%}Set {{ ledColor[0] | capitalize }} | void | Intensity: Number | Sets the brightness of the built-in LEDs so that they appear {{ ledColor[0] }}, using the specified intensity percentage (0-1).
+{{ ledColor[0] | capitalize }} On | void | None | Sets the brightness of the built-in LEDs so that they appear {{ ledColor[0] }} at full intensity.
```

Comment:
May be `builtin LEDs` should be replaced with `builtin EV3 LEDs` along with @dlech's comment earlier.